layouts/partials/js/expand.js

```diff
@@ -0,0 +1,95 @@
+// Setup all expandable tables on the page.
+var tables = document.querySelectorAll('.p-table--expanding')
```

Comment:
Please use `const` or `let` throughout this file - ideally `const`.

Reply:
Yeah, this was taken from vanillaframeworks examples, but I fixed it now!